lua/lvim/lsp/init.lua

```diff
@@ -120,6 +120,11 @@ function M.setup()
 
   set_handler_opts_if_not_set("textDocument/hover", vim.lsp.handlers.hover, { border = "rounded" })
   set_handler_opts_if_not_set("textDocument/signatureHelp", vim.lsp.handlers.signature_help, { border = "rounded" })
+
+  -- Enable rounded borders in :LspInfo window.
+  pcall(function()
+    require("lspconfig.ui.windows").default_options.border = "rounded"
+  end)
```

Comment:
```suggestion
   require("lspconfig.ui.windows").default_options.border = "rounded"
```
I think if `lspconfig` is loaded succesfully at this line below then we are ok without `pcall`
https://github.com/LunarVim/LunarVim/blob/7b413c428501b93e71e0342ae6d21c2b632c34ac/lua/lvim/lsp/init.lua#L92